src/main/java/org/folio/rest/service/report/RefundReportService.java

```diff
@@ -231,20 +233,29 @@ else if (actionIsOfType(feeFineAction, REFUND)) {
         reportEntry
           .withFeeFineType(account.getFeeFineType())
           .withBilledAmount(formatMonetaryValue(account.getAmount()))
-          .withDateBilled(formatDate(account.getMetadata().getCreatedDate(), ctx.timeZone));
+          .withDateBilled(formatDate(account.getMetadata().getCreatedDate(), ctx.timeZone))
+          .withFeeFineOwner(account.getFeeFineOwner());
       } else {
         log.error("Refund report - account is null, refund action {}", feeFineAction.getId());
       }
 
       if (accountCtx != null) {
+        if (isRefundedToPatron(feeFineAction)){
+          reportEntry.withPaidAmount(accountCtx.paidAmount.toString());
+        } else if (isRefundedToBursar(feeFineAction)) {
+          reportEntry.withTransferredAmount(accountCtx.transferredAmount.toString())
+            .withTransferAccount(singleOrDefaultMessage(accountCtx.transferAccounts,
+            MULTIPLE_MESSAGE));
+        } else {
+          reportEntry.withPaidAmount(accountCtx.paidAmount.toString())
+            .withTransferredAmount(accountCtx.transferredAmount.toString())
+            .withTransferAccount(singleOrDefaultMessage(accountCtx.transferAccounts,
+              MULTIPLE_MESSAGE));
```

Comment:
I think we can avoid duplicating code here if for our conditions above we used something like `isRefundedToPatron && ! isRefundedToBursar` and `isRefundedToBursar && ! isRefundedToPatron `